### PR TITLE
rocAL - fix for copy-write violation

### PR DESCRIPTION
**Pull request by rrawther**
This fixes the issue with copyright violation from palameda scan

### CODE REVIEW

**Comment by rrawther** [comment]
@LakshmiKumar23: can you please test this with the PYTHON_UNIT_TEST.sh

**Comment by LakshmiKumar23** [comment]
@rrawther I am seeing this error:
```
In while loop
n
In while loop
d
1
false

print_tensor
no-display
no-rocal-gpu
print_tensor
usage: rocAL_api_python_unittest.py [-h]
                                    [--image-dataset-path IMAGE_DATASET_PATH]
                                    [--batch-size BATCH_SIZE] [--display]
                                    [--print_tensor] [--classification]
                                    [--rocal-gpu] [--NHWC] [--fp16]
                                    [--local-rank LOCAL_RANK]
                                    [--world-size WORLD_SIZE]
                                    [--num-threads NUM_THREADS]
                                    [--num-epochs NUM_EPOCHS] [--seed SEED]
                                    [--augmentation-name AUGMENTATION_NAME]
                                    [--json-path JSON_PATH]
                                    [--detection DETECTION]
                                    [--video-path VIDEO_PATH]
                                    [--sequence-length SEQUENCE_LENGTH]
```

File: rocAL/rocAL_pybind/example/new_api/parse_config.py [hunk 1]
```diff
@@ -36,14 +11,17 @@ def parse_args():
...
-    add_bool_arg(common_group,'rocal-gpu', default=True)
-
-    common_group.add_argument('--NHWC', action='store_true',
+    common_group.add_argument('--display', default=False, action="store_true",
+                        help='--display:to display output from the pipeline')
+    common_group.add_argument('--print_tensor', default=False, action="store_true",
+                        help='--print_tensor: to print tensor output from the pipeline')
+    common_group.add_argument('--classification', default=False, action="store_true",
+                        help='--classification: to use for classification')
+    common_group.add_argument('--rocal-gpu', default=False, action="store_true",
```
Comment (LakshmiKumar23) [inline rocAL/rocAL_pybind/example/new_api/parse_config.py:20]: @rrawther Seems like this should default to "True" as per line 42 in old code

Comment (rrawther) [reply rocAL/rocAL_pybind/example/new_api/parse_config.py:20]: @LakshmiKumar23: I fixed the changes for this. Please recheck

File: rocAL/rocAL_pybind/run.sh [hunk 2]
```diff
@@ -50,5 +50,5 @@ else
   do
     echo "Going to install $WHEEL_NAME"
   done
-  pip$PYTHON_VERSION install $WHEEL_NAME
+  pip3 install $WHEEL_NAME
```
Comment (LakshmiKumar23) [inline rocAL/rocAL_pybind/run.sh:53]: @rrawther did you mean to check this in? In this PR?

Comment (LakshmiKumar23) [reply rocAL/rocAL_pybind/run.sh:53]: @rrawther The test shell script passes otherwise. 
@kiritigowda Can merge once above comment is resolved

Comment (rrawther) [reply rocAL/rocAL_pybind/run.sh:53]: reverting it

**Review by LakshmiKumar23 — APPROVED** [review]
@kiritigowda can be merged

**Comment by rrawther** [comment]
@kiritigowda Can you please merge this. Lakshmi has already tested it

**Pull request merged**